config/osu.php

```diff
@@ -21,6 +21,9 @@
         'thumbnailer' => env('BM_PROCESSOR_THUMBNAILER', 'http://localhost:4001'),
         'sentry' => env('BM_PROCESSOR_SENTRY'),
     ],
+    'beatmapset' => [
+        'required_hype' => get_int(env('BEATMAPSET_REQUIRED_HYPE')) ?? 5,
```

Comment:
why not just `get_int(env('BEATMAPSET_REQUIRED_HYPE', 5))`?

Reply:
because `BEATMAPSET_REQUIRED_HYPE=` will turn it into 0.

Reply:
good thing we used that pattern everywhere else in this file then 👍